### ISSUE TITLE
DP #49: Online Blockchain Tech Education 

### ISSUE THREAD

**Issue by Swailem95** [district0x/district-proposals#49]
## Name 

BlockSchool

## Purpose 

This district will allow people to enroll in courses that teach blockchain-related technologies. Someone could also set up, design and publish his own course that other people can enroll in for a predetermined fee. This district can also integrate with [ethlance](https://ethlance.com/) so that people can offer to work with their newly learned skills.  

# Description 

The purpose of this district is threefold. One of the most common problems companies have when hiring blockchain devs is actually finding good candidates that know what they're doing, are well-versed in the industry and understand the technology. This is partly because blockchain technology is fairly new it's developers are limited. Another purpose is that many people are interested in starting a career in blockchain technology and need to know how to start, what to learn and how to learn it. 

In a nutshell, this district will: 

- Allow people that are interested in blockchain technologies to learn about it (ie: Solidity and smart contracts).
- Allow people with knowledge in blockchain tech to teach others and get paid for it. 
- Allow companies to easily find blockchain developers as they increase. 

This is a win-win-win scenario. Companies find developers, developers offer their expertise to teach and people get to learn how blockchain technologies. Thus enlarging the ecosystem and spreading the knowledge around. 

Course curriculum will be left to the teacher to decide, students can check the curriculum and other course info before enrolling. Teachers can decide on how they want the course to operate (ie: number of students, duration, pricing, assignments, etc).

**Comment by diop** [comment]
This is an awesome idea. I would even call it the "School District". Start with Blockchain Education and then expand. I'll be the first Ethereum Blockchain Development Fundamentals instructor :) I don't know much. But I'll share what I know.

**Comment by Swailem95** [comment]
Nobody knows everything :)

**Comment by diop** [comment]
Haha agreed. I already started on a curriculum. I cannot wait to get started https://gist.github.com/diop/7c81acf501bdbd1336a17d0f846aa7b4

**Comment by Swailem95** [comment]
Very nice! I actually want to learn a lot of the topics you have written down. Especially React and the Dapp frameworks.

**Comment by diop** [comment]
Ok cool. It's actually getting easier to tie in Dapps and React these days. Check out TruffleJS [Truffle Boxes](http://truffleframework.com/boxes/react)

**Comment by diop** [comment]
I really hope that this School District gets approved. Education is the biggest asset this industry can invest in. Let's do it! 🥇

**Comment by Swailem95** [comment]
I hope so too. Looks like it needs 57 more upvotes to reach quorum :/ 
Voting process needs to change. No one is upvoting any of the proposals!

**Comment by always2late** [comment]
just suggested this exact idea in slack and was directed here. We should definitely plan this out right so that we can offer this as a legit accredited program

**Comment by Swailem95** [comment]
Definitely down. What are you slack usernames @always2late @diop?

**Comment by always2late** [comment]
Always2late
